wordpress-mobile/WordPress-iOS
# Issues/6189 preserve posts for restoration

Fixes #6189 
This PR applies the same trick used for previewed topics to topic-less posts. When saving state a flag is set indicating an orphaned post should be preserved and not cleaned up. When state is restored the flag is unset. 

To test:
Confirm the issue by viewing the full post detail of a Discover post and attempting to save and restore state. You should end back up on the discover feed.
Confirm the fix by switching to this branch and repeating the above steps. You should remain on the discover post.

Needs review: @frosty
